chat-plugins/info.js

```diff
@@ -1378,11 +1379,30 @@ exports.commands = {
 
 		if (!totalMatches) return this.errorReply("No " + (target ? "matched " : "") + "formats found.");
 		if (totalMatches === 1) {
-			let format = Dex.getFormat(Object.values(sections)[0].formats[0]);
+			let html = [];
+			let format = Dex.getFormat(targetId);
```

Comment:
Why did you change the value of `format` here? If you get a single match but it's not an exact match then you get a message about there being no banlist/ruleset.

Reply:
I was copy/pasting code from when it was a separate command. Should I just change it back?

Reply:
I think it would make sense to change it back, yes.